src/parser/mod.rs

```diff
@@ -2328,19 +2327,25 @@ impl<'a> Parser<'a> {
         }
     }
 
-    /// Bigquery specific: Parse a struct literal
     /// Syntax
     /// ```sql
-    /// -- typed
+    /// -- typed, specific to bigquery
```

Comment:
```suggestion
    /// -- typed
```
for the comments mentioning bigquery specific I think we can skip since the definition is now encoded in the dialect method. Otherwise it'll be easier to go out of sync if other dialects support it